### DISCUSSION TITLE
Using scss files requires me to install node-sass

### DISCUSSION THREAD

**Discussion by VladSez** [vercel/next.js#10937] [Help]
![image](https://user-images.githubusercontent.com/16666234/76327405-03e6eb80-62fb-11ea-8441-aa65cf2cd1ce.png)

Repo: https://github.com/VladSez/nextjs-sass-issue

**Comment by ItsCosmas** [comment]
Sorry but you actually have to install `node-sass` to work with `scss` in your app. At least that's all I know at the moment

**Reply by VladSez** [reply]
Thank you🙂

**Reply by timneutkens** [reply]
We recommend using `sass` instead of `node-sass`: https://nextjs.org/docs/basic-features/built-in-css-support#sass-support

**Comment by timneutkens** [comment]
We're going to make the message a lot clearer very soon, but yes you're required to:

```
npm install sass
```

We've updated the blogpost to reflect this too.

Documentation for the feature can be found here: https://nextjs.org/docs/basic-features/built-in-css-support#sass-support

**Reply by VladSez** [reply]
Thanks for clarification
